docs/pages/debugging/tools.mdx

```diff
@@ -205,11 +205,16 @@ Pressing the <kbd>shift</kbd> + <kbd>m</kbd> shortcut will also show installed [
 
 ## React Native Debugger
 
-> **warning** The React Native Debugger does not support apps using Hermes, see the [Chrome DevTools](#debugging-with-chrome-devtools) section.
+> **warning** The React Native Debugger requires Remote JS debugging, which has been deprecated since [React Native 0.73](https://reactnative.dev/docs/other-debugging-methods#remote-javascript-debugging-deprecated).
+> The React Native Debugger is a standalone app that wraps the React DevTools, Redux DevTools, and the Chrome DevTools. Unfortunately, [React Native Debugger requires](https://github.com/jhen0409/react-native-debugger/discussions/774) the deprecated Remote JS debugging workflow and is incompatible with Hermes.
 
-The React Native Debugger includes many tools listed later on this page, all bundled into one, including [React DevTools](#debugging-with-react-devtools) and network request inspection. For this reason, if you use one tool from this page, it should probably be this one.
+If you are using Expo **SDK 50** or **later**, you can use the [Expo Devtool plugins](/debugging/devtools-plugins) equivalents to the React Native Debugger.
```

Comment:
```suggestion
If you are using Expo **SDK 50** or **above**, you can use the [Expo dev tool plugins](/debugging/devtools-plugins) equivalents to the React Native Debugger:
```

- When this sentence ends with a period, as a reader, I'm not sure what to make of the list of items just below it. Let's end with colon to indicate which sections a user needs to look into (as alternatives). 
- Nit: "Expo dev tools plugins" is what we use throughout the documentation.

![CleanShot 2024-09-30 at 18 11 29@2x](https://github.com/user-attachments/assets/3b5d3b0b-d4c5-48ee-8cae-b2eec16e2c94)

Reply:
> When this sentence ends with a period, as a reader, I'm not sure what to make of the list of items just below it. Let's end with colon to indicate which sections a user needs to look into (as alternatives).

Good call

> Nit: "Expo dev tools plugins" is what we use throughout the documentation.

Sorry, I based this on the page title of the documentation page.

Reply:
no worries at all :)